packages/core/src/components/forms/radioGroup.tsx

```diff
@@ -124,15 +141,15 @@ export class RadioGroup extends AbstractPureComponent<RadioGroupProps> {
     }
 
     private getRadioProps(optionProps: OptionProps): Omit<RadioProps, "ref"> {
-        const { name } = this.props;
+        const { name, inline, disabled: groupDisabled, selectedValue, onChange } = this.props;
```

Comment:
Are the changes here functional or cleanup? If it's the latter, can we scope these changes into another PR?

Reply:
It's just a style change to remove the mix of destructuring and not destructuring `this.props` in this function.

Not worth the change, reverted.